2269/2/raper.rb

```diff
@@ -0,0 +1,36 @@
+# Class represents Info about Raper
+class BattleInfo
+  def initialize
+    @words_round ||= 0
+    @words_battle ||= 0
+    @bad_words ||= 0
+    @file_name ||= []
+  end
+
+  attr_reader :words_battle, :bad_words, :words_round, :file_name
+
+  def set_result(total, bad, round)
+    @words_battle += total
+    @bad_words += bad
+    @words_round = round
+  end
+
+  def file_source_info(file)
+    @file_name << file
+  end
+end
+
+# Class that represents each Rap Singer.
+class Raper
```

Comment:
having a class Raper is good, but is Battle entity valuable for the given task? should we simply count battles? or is it a distinct entity

btw raper == rapist, rapper == performer